cli/tests/unit_cli.yaml

```diff
@@ -20,9 +20,8 @@ services:
   cli:
     image: sawtooth-dev-test:$ISOLATION_ID
     volumes:
-      - ../../:/project/sawtooth-core
-    working_dir: /project/sawtooth-core/
-    entrypoint: nose2-3 -s /project/sawtooth-core/cli/tests
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    command: nose2-3 -v -s /project/sawtooth-core/cli/tests
```

Comment:
The working directory is now set to `/` in the docker containers, so I think the `-s` is necessary so nose2-3 doesn't search the whole file system. Although I haven't tested that.